Tests/Tests/DeviceMetaDataSpec.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+import Quick
+import Nimble
+import UIKit
+
+@testable import REventLogger
+
+class DeviceMetaDataSpec: QuickSpec {
+
+    override func spec() {
+        describe("check device info") {
+            let deviceInfoMock = DeviceInfoMock()
+            DeviceInfo.deviceBundle = deviceInfoMock
+
```

Comment:
Please add tests for device model and device name.

Reply:
These two parameters cannot be tested since deviceModel() returns different values for simulator and devices.